chart/kubeapps/templates/kubeapps-frontend-config.yaml

```diff
@@ -55,6 +57,20 @@ data:
         include "./server_blocks/k8s-api-proxy.conf";
       }
 
+      # Ensure each additional cluster can be reached (should only be
+      # used with an auth-proxy where k8s credentials are never leave
+      # the cluster). See additionalClusters option.
+      {{- range .Values.featureFlags.additionalClusters }}
+      location ~* /api/clusters/{{ .name }} {
+        rewrite /api/clusters/{{ .name }}/(.*) /$1 break;
+        rewrite /api/clusters/{{ .name }} / break;
+        proxy_pass {{ .apiServiceURL }};
+        proxy_ssl_trusted_certificate "./server_blocks/{{ .name }}-ca.pem";
```

Comment:
is the ca-cert mandatory?

Reply:
Good point, it shouldn't be mandatory as some api servers will have publicly signed certs. Done.